index.html

```diff
@@ -37,7 +37,7 @@
 
 <p>A <strong>Code of Ethics and Professional Conduct</strong> is useful to
 define accepted and acceptable behaviors and to promote high standards of
-Ralphprofessional practice. It also provides a benchmark for self evaluation and
+professional practice. The goal of this code is to ensure transparency in moderation of the organzition and to an environment where everyone can participate without fear of harassment. It also provides a benchmark for self evaluation and
```

Comment:
"moderation of the organization and to an environment" reads oddly. Maybe break into shorter sentences?